docs/advanced-usage/registry-management.md

```diff
@@ -135,3 +135,34 @@ dokku registry:set node-js-app push-on-release
 # globally
 dokku registry:set --global push-on-release
 ```
+
+### Push extra tags 
+
+To push the image on release with extra tags, set the `push-extra-tags` to a comma-separated list of tags via the `registry:set` command. The default value for this property is `false`. Setting the property will result in the image being tagged with extra tags every release.
```

Comment:
The default is an empty string?

Reply:
Yes, as per: https://github.com/dokku/dokku/pull/6588/files#diff-3b33c1b424a2329370c53d6d235d92e5032e63f0ed15b824f88c87f83baa6f28R9

Reply:
Mind updating the docs to not say `false`, but instead say it's empty or something?